clients/src/main/java/org/apache/kafka/common/requests/ConsumerGroupHeartbeatRequest.java

```diff
@@ -73,6 +73,7 @@ public AbstractResponse getErrorResponse(int throttleTimeMs, Throwable e) {
             new ConsumerGroupHeartbeatResponseData()
                 .setThrottleTimeMs(throttleTimeMs)
                 .setErrorCode(Errors.forException(e).code())
+                .setErrorMessage(e.getMessage())
```

Comment:
I suggest to use `ApiError` as follow:
```
ApiError apiError = ApiError.fromThrowable(exception)
```
Then, you can set error message based on `apiError`.